staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -663,7 +663,7 @@ func TestCacherNoLeakWithMultipleWatchers(t *testing.T) {
 				ctx, _ := context.WithTimeout(context.Background(), 3*time.Second)
 				w, err := cacher.Watch(ctx, "pods/ns", storage.ListOptions{ResourceVersion: "0", Predicate: pred})
 				if err != nil {
-					t.Fatalf("Failed to create watch: %v", err)
+					t.Errorf("Failed to create watch: %v", err)
```

Comment:
doing this just makes it likely that you're going to NPE on the `w` next line down, right?  Seems like you're missing a return

Reply:
> doing this just makes it likely that you're going to NPE on the `w` next line down, right? Seems like you're missing a return

Thanks , I fixed it

Reply:
I'm not clear on why a continue is better in this case than a return.  You know this has failed and the continue just re-loops you for no particular gain, right?